mitodl/open-discussions
# remove runs from videos and podcasts

#### Pre-Flight checklist

#### What are the relevant tickets?
closes https://github.com/mitodl/open-discussions/issues/3022

#### What's this PR do?
We created blank runs for podcasts and videos so that the price filter would work. Now that that filter is removed podcasts and videos don't need runs

#### How should this be manually tested?
run
docker-compose run web ./manage.py backpopulate_youtube_data fetch
and
docker-compose run web ./manage.py backpopulate_podcast_data and verify that neither throw any errors.
